chart/values.yaml

```diff
@@ -1,3 +1,8 @@
+ssmPath: /dsva-vagov/vets-api/dev/
```

Comment:
You can leave these values blank and update in the manifest repo, [here](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/dev/values.yaml)

Reply:
one of the goals here is to push the values into the parent helm chart so that we have consistency between environments.  The long term plan is to minimize / eliminate the 'manfest' helm charts. 

Please se:
https://vfs.atlassian.net/wiki/spaces/PRT/pages/3520462849/vets-api+Helm+chart+normalization+-+Design+Document
https://vfs.atlassian.net/wiki/spaces/PRT/pages/3520987139/vets-api+helm+chart+into+master+branch+-+Design+Document

Reply:
Thank you for clarifying